snomed/com.b2international.snowowl.snomed.api.rest.tests/src/com/b2international/snowowl/snomed/api/rest/components/SnomedModuleDependencyRefsetTest.java

```diff
@@ -0,0 +1,115 @@
+			);
+		});
+		
+		// create Norwegian module
+		Map<?, ?> norwegianModuleRequestBody = createConceptRequestBody(Concepts.MODULE_ROOT, NORWEGIAN_MODULE_CONCEPT_ID, SnomedApiTestConstants.UK_PREFERRED_MAP)
+				.put("id", NORWEGIAN_MODULE_CONCEPT_ID)
+				.put("commitComment", "Created norwegian module concept")
+				.build();
+		createComponent(branchPath, SnomedComponentType.CONCEPT, norwegianModuleRequestBody).statusCode(201);
+		
+		// create both inferred and stated relationships
+		Map<?, ?> inferredRelationshipRequestBody = SnomedRestFixtures
+				.createRelationshipRequestBody(NORWEGIAN_MODULE_CONCEPT_ID, Concepts.IS_A, Concepts.MODULE_ROOT, NORWEGIAN_MODULE_CONCEPT_ID, CharacteristicType.INFERRED_RELATIONSHIP, 0)
+				.put("commitComment", "Created inferred is_a from the norwegian rule to SCT_MODULE_CORE").build();
+		
+		Map<?, ?> statedRelationshipRequestBody = SnomedRestFixtures
+				.createRelationshipRequestBody(NORWEGIAN_MODULE_CONCEPT_ID, Concepts.IS_A, Concepts.MODULE_ROOT, NORWEGIAN_MODULE_CONCEPT_ID, CharacteristicType.STATED_RELATIONSHIP, 0)
+				.put("commitComment", "Created state is_a from the norwegian rule to SCT_MODULE_CORE").build();
+
+		createComponent(branchPath, SnomedComponentType.RELATIONSHIP, inferredRelationshipRequestBody).statusCode(201);
+		createComponent(branchPath, SnomedComponentType.RELATIONSHIP, statedRelationshipRequestBody).statusCode(201);
+		
+		SnomedConcepts concepts = SnomedRequests.prepareSearchConcept()
+				.filterById(NORWEGIAN_MODULE_CONCEPT_ID)
+				.build(SnomedDatastoreActivator.REPOSITORY_UUID, branchPath.getPath())
+				.execute(getBus())
+				.getSync();
+
+		concepts.forEach(c -> assertEquals("Effective time must still be null", null, c.getEffectiveTime()));
+				
+		// version branch
+		final String effectiveDate = getNextAvailableEffectiveDateAsString(SnomedTerminologyComponentConstants.SNOMED_SHORT_NAME);
+		final String versionId = "testForModuleDependencyMembers";
+		createVersion(shortName, versionId, effectiveDate).statusCode(201);
+		getVersion(shortName, versionId).statusCode(200);
+
+		SnomedReferenceSetMembers moduleDependencyMembersAfterVersioning = SnomedRequests.prepareSearchMember()
+				.all()
+				.filterByRefSet(Concepts.REFSET_MODULE_DEPENDENCY_TYPE)
+				.build(SnomedDatastoreActivator.REPOSITORY_UUID, branchPath.getPath())
+				.execute(getBus())
+				.getSync();
+		
+		moduleDependencyMembersAfterVersioning.getItems().forEach(member-> {
+			final Pair<String, String> pair = Tuples.pair(member.getModuleId(), member.getReferencedComponent().getId());
+			final Date originalMemberEffectiveTime = moduleToReferencedComponentAndEffectiveDateMap.get(pair);
+			final Date versionDate = EffectiveTimes.parse(effectiveDate, DateFormats.SHORT);
+			if(originalMemberEffectiveTime != null) {
+				assertEquals("Effective dates on existing module dependency members shouldn't be updated after versioning" , originalMemberEffectiveTime,  member.getEffectiveTime());
```

Comment:
Effective time shouldn't be updated for INT members (where both the moduleId and the referenced component id is an international module). This is not equivalent with what you wrote here!